Application/EdFi.Ods.Api/Startup/OdsStartupBase.cs

```diff
@@ -270,7 +271,7 @@ void SetStaticResolvers()
 
         private void LoadPlugins()
         {
-            var pluginFolder = Plugin.Folder;
+            var pluginFolder = Path.GetFullPath(Plugin.Folder);
```

Comment:
GetFullPath method is already called in PluginFinder.FindAssemliesWithPlugins.  Please remove.